src/test/rustdoc/bad-codeblock-syntax.rs

```diff
@@ -1,42 +0,0 @@
-// @has bad_codeblock_syntax/fn.foo.html
```

Comment:
Can this file be re-added? It is partly making sure rustdoc doesn't crash when attempting to syntax highlight invalid rust.